.azure-pipelines/azure-pipelines-windows.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       llvm.version: '11.0.0'
       mkl.version: '2019.1'
       cmake.build_type: 'Release'
-      conda.build: false
+      conda.build: true
```

Comment:
What does the `conda.build` line do? Based on commit history, it looks like something with the OpenMP copy, but I want to make sure this is still necessary. Will approve once that's cleared up.

Reply:
the `conda.build` line is separate from the omp business. the boolean activates an extra step in the azure (all set up by Raimondas) to build the Windows conda package, spot test, and save the result as an artifact. I toggled it here to make sure the omp lib changes weren't scuppering the conda package, but you're right that it should be off again in general (adds ~25 min, iirc). But I'm putting off changing it until I see if there are merge conflicts to resolve at the same time.